aten/src/ATen/native/cuda/BatchLinearAlgebraLib.cu

```diff
@@ -1223,6 +1251,37 @@ void linalg_eigh_cusolver(Tensor& eigenvalues, Tensor& eigenvectors, Tensor& inf
   }
 }
 
+void lu_solve_looped_cusolver(const Tensor& b, const Tensor& lu, const Tensor& pivots) {
+  AT_DISPATCH_FLOATING_AND_COMPLEX_TYPES(b.scalar_type(), "lu_solve_cusolver", [&] {
+    int n = cuda_int_cast(lu.size(-2), "n");
+    int nrhs = cuda_int_cast(b.size(-1), "nrhs");
+    auto batch_size = batchCount(lu);
+    auto infos = at::zeros(batch_size, lu.options().dtype(kInt));
+    auto infos_data = infos.data_ptr<int>();
```

Comment:
`info` argument to `cusolverDn<t>getrs` only indicates if other arguments are incorrect. We don't need `batch_size` of them, only one.
```suggestion
    auto info = at::zeros({1}, lu.options().dtype(kInt));
    auto info_data = infos.data_ptr<int>();
```